components/constellation/constellation.rs

```diff
@@ -5396,24 +5472,25 @@ where
     }
 
     /// Send the frame tree for the given webview to the compositor.
-    fn update_frame_tree_if_focused(
-        &mut self,
-        top_level_browsing_context_id: TopLevelBrowsingContextId,
-    ) {
-        // Only send the frame tree if the given webview is focused.
-        if let Some(focused_webview_id) = self.webviews.focused_webview().map(|(id, _)| id) {
-            if top_level_browsing_context_id != focused_webview_id {
-                return;
+    fn update_webview(&mut self, top_level_browsing_context_id: TopLevelBrowsingContextId) {
```

Comment:
I think `send_frame_tree_if_focused()` is a bit more descriptive of what this function is actually doing. "Update" is so generic.

Reply:
Renamed to `send_frame_tree_if_focused` in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)